src/qt/trafficgraphwidget.cpp

```diff
@@ -128,8 +128,9 @@ void TrafficGraphWidget::updateRates()
 
     quint64 bytesIn = clientModel->node().getTotalBytesRecv(),
             bytesOut = clientModel->node().getTotalBytesSent();
-    float inRate = (bytesIn - nLastBytesIn) / 1024.0f * 1000 / timer->interval();
-    float outRate = (bytesOut - nLastBytesOut) / 1024.0f * 1000 / timer->interval();
+    // Rates are in bytes/millisecond which is equal to kilobytes/second
+    float inRate = (bytesIn - nLastBytesIn) * 1.0f / timer->interval();
+    float outRate = (bytesOut - nLastBytesOut) * 1.0f / timer->interval();
```

Comment:
Why `  * 1.0f` is required?

Reply:
I am no C++ expert, just took it over from `/ 1024.0f * 1000` before. If the `1024.0f` was also not needed (I imagined it might have had some reason of C++ type conversion), I will remove the `* 1.0f`

Reply:
If type conversion is supposed, it would better to apply explicit conversion.